pytest_inmanta/plugin.py

```diff
@@ -68,6 +67,15 @@
 )
 from .test_parameter import ParameterNotSetException, TestParameterRegistry
 
+try:
+    """
+    Those classes are only used in type annotation, but the import doesn't work
+    in python 3.6.  So we simply catch the error and ignore it.
+    """
+    from pytest import CaptureFixture, OptionGroup, Parser
```

Comment:
Don't you need to quote their usage if you only import them conditionally?

Reply:
That is indeed the case